src/index.ts

```diff
@@ -0,0 +1,49 @@
+import { Express, Request, Response } from 'express';
+import { UserValidation } from '../resources/UserValidation';
+
+const express = require('express');
+const app: Express = express();
+require('dotenv').config();
+
+app.use(express.json());
+
+const user = new UserValidation;
+
+//request users data for testing
+app.get('/users', (req: Request, res: Response) => {
+    res.json(user.users);
+})
+
+app.post('/users', async (req: Request, res: Response) => {
+    const error = user.registerValidation(req.body.email, req.body.password);
+    const thereIsError = error != "";
+    if (thereIsError) {
+        return res.status(400).send(error);
+    }
+    try {
+        user.register(req.body.email, await user.hashPassword(req.body.password));
```

Comment:
It does seem a bit odd that UserValidation is handling actions like register or login. What responsibilities do you think UserValidation should have?

Reply:
It should check the validity of the email and password that were input, fixed this as well